### PR TITLE
AP-5269: Update delete application handling

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-5269)

Remove the modal JS dialog and update the non-js path to match the requirement.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: features/providers/deleting_applications.feature [hunk 1]
```diff
@@ -1,34 +1,15 @@
 Feature: Deleting applications
 
   @javascript
```
Comment (jsugarman) [inline features/providers/deleting_applications.feature:3]: does feature work pass without the javascript tag?

**Review by jsugarman — APPROVED** [review]
LGTM. Are we using an existing delete application page then

**Comment by colinbruce** [comment]
> LGTM. Are we using an existing delete application page then

Yes, there was always a standard delete page.  The JS version git added over the top

File: features/providers/deleting_applications.feature (hunk 1 above)
Comment (colinbruce) [reply features/providers/deleting_applications.feature:3]: No, the features hang at the first step without the tag! 🤷‍♂️

**Pull request merged**